Interop/Runtime/src/main/kotlin/kotlinx/cinterop/StableRef.kt

```diff
@@ -26,17 +26,16 @@ typealias StableObjPtr = StableRef<*>
  *
  * Any [StableRef] should be manually [disposed][dispose]
  */
-data class StableRef<out T : Any> @PublishedApi internal constructor(
-        private val stablePtr: COpaquePointer,
-        private val ref: T // Note: storing the reference itself to avoid unchecked casts.
+inline class StableRef<out T : Any> constructor(
```

Comment:
The constructor can be made internal again using `@Suppress("NON_PUBLIC_PRIMARY_CONSTRUCTOR_OF_INLINE_CLASS")`
(it is often applied to stdlib classes, e.g. `Result`, `UInt` etc.)